test/test_helpers/form_select_assertions.rb

```diff
@@ -98,6 +98,16 @@ def assert_select_no_button_to(*args)
 
       assert_select_button_to(*args)
     end
+
+    # Matcher to check that a specific element does not exist
+    #
+    # == Example
+    #
+    #  refute_select '#some_id'
+    #
+    def refute_select(selector, args = {})
```

Comment:
I think this is too generic and does not really strongly assert or refute anything.
If you change the code CSS a bit will still be green so not enough value.
I think tests should be strong enough to protect from mutation.

Reply:
@hallelujah any idea? 🤔

Reply:
@hallelujah what about:

```
  def refute_select(*args)
    begin
      assert_select(*args)
      return false
    rescue AssertionFailedError
      return true # there is no existing field
    end
  end
end
```

Reply:
Why refute select?
Can we do assert select instead?
Otherwise I think it is not worth it because anyone can change it from select to equal and it will still pass

Reply:
I wanted to test that the select didn't have the `Please select` value